flash_notebooks/finetuning/image_classification.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p>The trainer object can be used for training or fine-tuning tasks on new sets of data. </p><p> </p><p>You can pass in parameters to control the training routine- limit the number of epochs, run on GPUs or TPUs, etc.</p><p>For more details, read the Trainer documentation (add link).</p><p> </p><p>In this demo, we will limit the fine-tuning to run just one epoch using <code>max_epochs=1</code>.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>